packages/neo-one-typescript-concatenator/src/Concatenator.ts

```diff
@@ -518,4 +570,51 @@ export class Concatenator {
 
     return alias === undefined ? node : alias;
   }
+
+  private createRenamedImport(name: ts.Identifier, symbols: ReadonlyArray<ts.Symbol>) {
+    return ts.createVariableStatement(
+      undefined,
+      ts.createVariableDeclarationList(
+        [
+          ts.createVariableDeclaration(
+            name,
+            undefined,
+            ts.createObjectLiteral(
+              symbols.reduce((acc: ReadonlyArray<ts.ObjectLiteralElementLike>, symbol) => {
+                const alias = tsUtils.symbol.getAliasedSymbol(this.context.typeChecker, symbol);
+                const expression =
+                  alias === undefined
+                    ? tsUtils.symbol.toExpression(this.context.typeChecker, symbol)
+                    : tsUtils.symbol.toExpression(this.context.typeChecker, alias);
+                if (expression === undefined) {
+                  return acc;
+                }
+
+                return acc.concat(ts.createPropertyAssignment(tsUtils.symbol.getName(symbol), expression));
```

Comment:
`ts.createPropertyAssignment(tsUtils.symbol.getName(symbol), this.getIdentifierForSymbol(symbol))`

Reply:
(You'll need to implement `getIdentifierForSymbol`, but the pieces are already there)